templates/advantage/_stripe-modal.html

```diff
@@ -46,18 +33,40 @@ <h2 class="p-modal__title js-contract-name u-no-margin--bottom" id="modal-title"
 
           <div class="p-form__group p-form-validation row u-no-padding u-vertically-center">
             <div class="col-3">
-              <label class="u-no-padding--top" for="name">Name on card:</label>
+              <label class="u-no-padding--top" for="email">Email my receipt to:</label>
             </div>
 
             <div class="col-9">
-              <input class="p-form-validation__input" name="name" type="text" required value="{{ user_info.fullname }}" />
+              <input class="p-form-validation__input" name="email" type="email" required value="{{ user_info.email }}" />
             </div>
 
             <div class="col-9 col-start-large-4">
               <span class="p-form-validation__message u-hide"></span>
             </div>
           </div>
 
+          {% if account is defined %}
+            <div class="p-form__group p-form-validation row u-no-padding u-vertically-center">
+              <div class="col-3">
+                <label class="u-no-padding--top" for="name">Account name:</label>
```

Comment:
Regardless of any future layout changes, I think “Account name:” here needs to be “Choose an account name:”. Otherwise, no matter what caption we used, it would look like something you’re supposed to know already when you don’t.

Unfortunately, in the Ubuntu font, “Choose an account name:” doesn’t fit inside               `<div class="col-3">`. I don’t know why we don’t use `<table>` for this kind of thing, but from quick experimenting I think the dialog would look fine with all the `col-3` changed to `col-4` and all the `col-9` changed to `col-8`.

Reply:
Done, "Choose account name" fits even without the column changes. "Account name" is shown if the account is already set.